submissions/sprint_b/__init__.py

```diff
@@ -0,0 +1,13 @@
+def configure_app():
+    import os
+
+    import dotenv
+    from flask import Flask, jsonify
+
+    PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
+    dotenv.load_dotenv(os.path.join(PROJECT_ROOT, '.env'))
+    app = Flask(__name__)
+    return app, jsonify
```

Comment:
configure_app method should return an app instance. jsonify can be imported in modules and does not need to be included here.